eng/pipelines/preview.yml

```
@@ -68,6 +68,13 @@ extends:
         $newContent | Set-Content $path -NoNewline
       displayName: 'Update Packages.Data.props'
       workingDirectory: $(sdkRepositoryPath)
+    AutorestInitializationSteps:
+    - ${{ if eq(parameters.UseTypeSpecNext, 'true') }}:
+      - task: Bash@3
+        inputs:
+          targetType: 'inline'
+          script: 'echo "##vso[build.addbuildtag]typespec_next"'
```

I would use powershell for xplat support

```suggestion
      - pwsh: |
          echo "##vso[build.addbuildtag]typespec_next"
```